templates/haproxy.cfg.j2

```diff
@@ -1,41 +1,35 @@
 global
```

Comment:
on the header of a file may add `{{ ansible_managed | comment }}` for remind ansible is manged.

Reply:
done